.github/workflows/sycl-detect-changes.yml

```diff
@@ -22,13 +22,13 @@ jobs:
         uses: actions/github-script@v7
         with:
           script: |
-            console.log("Number of files changed:");
-            console.log(context.payload.pull_request.changed_files);
-            return context.payload.pull_request.changed_files ;
+            const changedFiles = context.payload.pull_request.changed_files;
+            console.log("Number of files changed:", changedFiles);
```

Comment:
Is the `console.log` intended or just a leftover debug statement?

Reply:
It was intended -- as debug output of the step.